microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension.md

```diff
@@ -5,7 +5,7 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 07/07/2022
+ms.date: 09/07/2022
 ---
 # Microsoft Edge DevTools extension for Visual Studio Code
```

Comment:
@captainbrosset @codepo8 @JasonAndrewWriter - Not sure why the target PR has a general build error, but here's a Writer pass on the incoming h4 section, to merge into PR 2156.